### ISSUE TITLE
Can't import an .ics file exported from Evolution, Java errors

### ISSUE THREAD

**Issue by ioogithub** [SimpleMobileTools/Simple-Calendar#1077]
When I try to import an ics Canendar that was exported from Evolution, I get an error message:

An Error occurred:
java.lang.IllegalArgumentException: Invalid format: ""

After sometime it says "Import successful"

It appears to import some of the events but not all, maybe 30%.

I tried another test and created a Calendar in Evolution with 2 events. All defaults, I just created "Test event 1" and "Test event 2" and saved them. I tried to import this simple .ics file and although it said Import successful it didn't actually import the events- they don't show up.

I upgraded my version of Evolution, same results. I have tried importing to the Simple-Calendar and to the system calendar (CalDav).

I then tried using two other apps found on F-Droid to import the same files- Offline Calendar and Calendar Import-Export. Using these two apps I can successfully import my >ics files without errors, all events show up as expected.

The problem is that Etar, Lineage default Calendar app can see all of the events but Simple Calendar cannot see any events when I use this method. When I try to refresh it just spins for 5 or 6 minutes and gives up. It does see the Calendar that was created under CalDav Sync and the app storage grows to around 12mb but no events are shown.

I would like to use Simple Calendar but I can't import directly inside the app and if I import with other tools Simple Calendar doesn't display events when all of the other Calendars do.

Evolution version: 3.34.1-3
System: Debian Buster
System: LineageOS 16 nightly
Simple-Calendar 6.8.3 from F-Droid

**Comment by ioogithub** [comment]
I did some more testing and found something further.

One issue I found:

If I double click to create an event in Evolution, it will default to creating an event from 9am-9pm. If give it a title and save the event, Simple Calendar will not import this type of event. If I set the time manually, it will also not import these events.

If I check the "All day event box" it will import this type of event.

This doesn't address the illegal format error that is the main problem but it does address the simple test I described earlier.

**Comment by tibbi** [comment]
paste here some sample events .ics content please, so that I can reproduce it. It is a casual text file, you can open it with any text editor

**Comment by ioogithub** [comment]
I mentioned 3 issues:
1. Java error
2. Events with times don't appear
3. Events imported with 3rd party tool don't appear.

For #2, here is the simple test export I mentioned. There are 3 events. The two events with start and stop times do not show up, the one with no time (all day event) does show up as expected. ics renamed to txt due to github limitation.
[3.txt](https://github.com/SimpleMobileTools/Simple-Calendar/files/4172971/3.txt)

For #3 I have an update as well. I just tried another import using org.sufficientlysecure.localcalendar and org.sufficientlysecure.ical. Before I mentioned that I could see the events in etar and in the ASOP cal right away but not in Simple Cal however I do see events now after a reboot. I will test further. Any ideas why they show up immediately in the other cals but I need to reboot to see them in Simple Cal?

#1 is still the main issue but not sure how to troubleshoot it has 600 events going back 15 years. It is 300k is there a size or buffer limit? It imports and exports fine in Evolution, and sufficientlysecure.ical, not sure how to find which events Simple Cal doesn't like though.

**Comment by tibbi** [comment]
ok thanks, will take a look at it soon

**Comment by ioogithub** [comment]
I have an update for #3. I imported a calendar and when I go to Settings->CalDAV sync and select it, I see Syncing... then it does nothing. I wait for around 4 or 5 minutes and events do not show up.

The strange thing is, if I go to Etar and toggle that calendar there (Under Settings-> Local Calendars) it then shows up in Simple Calendar right away!? I can reproduce this every time! I am not sure how toggling a calendar to display in a totally separate app affects your app? 

I tried the same test with the ASOP calendar, if I enable it in Simple Calendar and wait nothing happens. If I select "Calendars to display" in the ASOP app, then the events again show up right away in Simple Calendar.

Perhaps there is some trigger that is being missed under Settings->CalDAV sync when a cal in enabled? I don't think it should be a performance issue is I can toggle the same calendar off and on and see an immediate result in the other apps.

Anyway I would prefer not to have all of these other apps to get things working. I really just want to use Simple Calendar.
